test/CNStakedKLAY.ts

```diff
@@ -568,6 +576,21 @@ describe('CNStakedKLAY', () => {
         });
       });
 
+      describe('#unstakeAll()', () => {
+        it('should unstake all balances', async () => {
```

Comment:
do we have coverage on `unstakeAll()` sweeps reward and still works as expected??